src/plephinteg.jl

```diff
@@ -90,111 +88,222 @@ end
 # end
 
 @doc raw"""
-    taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}}, 
-                        xaux::Vector{Taylor1{U}}, abstol::T, params, 
-                        parse_eqs::Bool=true) where {T<:Real, U<:Number}
+    taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}}, 
+                        abstol::T, params, parse_eqs::Bool=true) where {T<:Real, U<:Number}
+    taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}}, abstol::T, params,
+                        rv::TaylorIntegration.RetAlloc{Taylor1{U}}) where {T<:Real, U<:Number}
 
 Threaded version of `TaylorIntegration.taylorstep`.
 
 See also [`stepsize_threads`](@ref) and [`TaylorIntegration.taylorstep`](@ref).
 """
-function taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}},
-        dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}}, abstol::T, params,
-        parse_eqs::Bool=true) where {T<:Real, U<:Number}
+function taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}}, 
+                             abstol::T, params) where {T<:Real, U<:Number}
 
     # Compute the Taylor coefficients
-    TaylorIntegration.__jetcoeffs!(Val(parse_eqs), f!, t, x, dx, xaux, params)
-    # @time TaylorIntegration.__jetcoeffs!(Val(parse_eqs), f!, t, x, dx, xaux, params)
-    # @time TaylorIntegration.__jetcoeffs!(Val(false), f!, t, x, dx, xaux, params)
-    # @time TaylorIntegration.__jetcoeffs!(Val(true), f!, t, x, dx, xaux, params)
+    TaylorIntegration.__jetcoeffs!(Val(false), f!, t, x, dx, xaux, params)
+
+    # Compute the step-size of the integration using `abstol`
+    δt = stepsize_threads(x, abstol)
+
+    return δt
+end
+
+function taylorstep_threads!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Taylor1{U}}, abstol::T, params,
+                             rv::TaylorIntegration.RetAlloc{Taylor1{U}}) where {T<:Real, U<:Number}
+
+    # Compute the Taylor coefficients
+    TaylorIntegration.__jetcoeffs!(Val(true), f!, t, x, dx, params, rv)
 
     # Compute the step-size of the integration using `abstol`
     δt = stepsize_threads(x, abstol)
-    # δt = stepsize_jz05(x, abstol)
 
     return δt
 end
 
 @doc raw"""
-    taylorinteg_threads(f!, q0::Array{U,1}, t0::T, tmax::T, order::Int, abstol::T,
-                        params = nothing; maxsteps::Int=500, parse_eqs::Bool=true, 
-                        dense::Bool=false) where {T<:Real, U<:Number}
+    __determine_parsing!(parse_eqs::Bool, f, t, x, dx, params)
 
-Threaded version of `TaylorIntegration.taylorinteg`.
+Specialized method of `TaylorIntegration._determine_parsing!` to avoid invalidations.
```

Comment:
`SnoopCompile` detected invalidations triggered by the `methodswith` function within `TaylorIntegration._determine_parsing!`. Since `src/jetcoeffs.jl` contains the code generated by `@taylorize`, we can skip checking that the specialized methods exist.